bin/upgrade

```diff
@@ -160,8 +160,25 @@ function handle_image_upgrade() {
   set_server_pro_image_name "$SEED_IMAGE_VERSION"
   docker pull "$IMAGE"
   if [[ $GIT_BRIDGE_ENABLED == "true" ]]; then
-    set_git_bridge_image_name "$SEED_IMAGE_VERSION"
-    docker pull "$GIT_BRIDGE_IMAGE"
+    if [[ -n ${GIT_BRIDGE_IMAGE:-} ]]; then
+      echo "-------------------  WARNING  ----------------------"
+      echo "  You're using a custom git bridge image: $GIT_BRIDGE_IMAGE"
+      echo "  Upgrade of your custom image is skipped. You need to pull the image separately, making sure that:"
+      echo "     1. The Docker image is tagged with the new version: $SEED_IMAGE_VERSION"
+      echo "     2. The environment variable GIT_BRIDGE_IMAGE only contains the image name, and not a tag/version."
+      echo "  You can safely proceed with the upgrade process, but if you try subsequently to restart your instance"
+      echo "  it will fail until the updated image with tag $SEED_IMAGE_VERSION is available in your system"
+      echo "-------------------  WARNING  ----------------------"
+      local should_continue="n"
+      read -r -p "Continue with the upgrade process? [y/n] " should_continue
+      if [[ ! "$should_continue" =~ [Yy] ]]; then
+        echo "Exiting."
+        exit 1
+      fi
```

Comment:
```suggestion
      prompt "Continue with the upgrade process?"
```
We've got a helper for this :)

https://github.com/overleaf/toolkit/blob/e611552228b52fb5b631dbed4644004805700972/lib/shared-functions.sh#L125-L129

Reply:
ah nice!